cloud-architecture/custom-apps-and-consolidation/mysql-mds/files/terraform.tfvars

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
+# Purpose: terraform variable definition file.
+# @author: Vijay Kokatnur
+
+# Identity and access parameters
+home_region     = "eu-frankfurt-1"
+region          = "eu-frankfurt-1"
+
+
+# general oci parameters
+compartment_id = "ocid1.compartment.oc1.."
+label_prefix   = "none"
+
+# ssh keys
+ssh_private_key_path = "keys/id_rsa"
+# ssh_public_key       = ""
+# ssh_public_key_path  = "none"
+ssh_public_key_path = "keys/id_rsa.pub"
+
+#networking parameters
+subnet_id     = "ocid1.subnet.oc1..."
```

Comment:
Use placeholder.

Reply:
subnet_id     = "ocid1.subnet.oc1..." This is already a placeholder. Any other suggested format?